src/Bkwld/Croppa/Handler.php

```diff
@@ -104,6 +104,50 @@ public function handle($request) {
 
 	}
 
+
+	/**
+	 * Render image directly
+	 *
+	 * @param string $imagepath The imagepath!
+	 * @return boolean
+	 */
+	public function render($imagepath) {
```

Comment:
I think it makes sense for the new logic to exist in `Handler`, like you have it here, but I don't like how this duplicates so much of the code from `handle()`.  I think the code that `render()` shares with `handle()` should be DRYed up to it's own method, like `renderFromPath($path)` or something.